### ISSUE TITLE
Functional Layers won't get summarised

### ISSUE THREAD

**Issue by franz101** [sksq96/pytorch-summary#120]
`import torch
import torch.nn as nn
import torch.nn.functional as F
import torch.optim as optim
from torchvision import models
from torchsummary import summary

class Net(nn.Module):
    def __init__(self):
        super(Net, self).__init__()
        self.conv1 = nn.Conv2d(1, 32, 5, 1)
        self.conv2 = nn.Conv2d(32, 64, 5, 1)
        self.dropout1 = nn.Dropout2d(0.4)
        self.dropout2 = nn.Dropout2d(0.5)
        self.fc1 = nn.Linear(2048, 1024)
        self.fc2 = nn.Linear(1024, 10)

    def forward(self, x):
        x = self.conv1(x)
        print(x.size())
        x = F.relu(x)
        print(x.size())
        x = F.max_pool2d(x, 2,2)
        print(x.size())
        x = self.conv2(x)
        print(x.size())
        x = F.relu(x)
        print(x.size())
        x = F.max_pool2d(x, 2,2)
        print(x.size())
        x = x.view(-1,2048)
        x = self.fc1(x)
        x = F.relu(x)
        x = self.dropout1(x)
        x = self.fc2(x)
        output = F.log_softmax(x, dim=1)
        return output

device = torch.device('cuda' if torch.cuda.is_available() else 'cpu')
model = Net().to(device)
summary(model, (1,28, 28))`

OUTPUT:
`torch.Size([2, 32, 24, 24])
torch.Size([2, 32, 24, 24])
torch.Size([2, 32, 12, 12])
torch.Size([2, 64, 8, 8])
torch.Size([2, 64, 8, 8])
torch.Size([2, 64, 4, 4])
----------------------------------------------------------------
        Layer (type)               Output Shape         Param #
================================================================
            Conv2d-1           [-1, 32, 24, 24]             832
            Conv2d-2             [-1, 64, 8, 8]          51,264
            Linear-3                 [-1, 1024]       2,098,176
         Dropout2d-4                 [-1, 1024]               0
            Linear-5                   [-1, 10]          10,250
================================================================
Total params: 2,160,522
Trainable params: 2,160,522
Non-trainable params: 0
----------------------------------------------------------------
Input size (MB): 0.00
Forward/backward pass size (MB): 0.19
Params size (MB): 8.24
Estimated Total Size (MB): 8.43
----------------------------------------------------------------`

EXPECTED OUTPUT:
The Max Pooling Layers should be mentioned...

What am I missing here?

**Comment by TylerYep** [comment]
Hmm... I will look into this problem, but you may want to repost this issue on my fork - see #124
